test/core/end2end/gen_build_yaml.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.7
+#!/usr/bin/env python
```

Comment:
Nit: `python` explicitly refers to Python 2 on Posix systems, making this change effectively a no-op. Maybe we should just remove the shebang entirely?

Reply:
Shebang removed.